content/sensu-go/5.13/sensuctl/reference.md

```diff
@@ -689,6 +690,37 @@ cat .config/sensu/sensuctl/cluster
 
 These are useful if you want to know what cluster you're connecting to, or what namespace or username you're currently configured to use.
 
+## Interacting with Bonsai
+
+Sensuctl supports both installing asset definitions directly from [Bonsai](34) and checking your Sensu backend for outdated assets.
+
+### Usage
+
+To install an asset definition directly from Bonsai, use `sensuctl asset add [NAME][:VERSION]`, replacing `[NAME]` with the name of the asset from Bonsai. `[:VERSION]` is only required if you require a specific version or are pinning to a specific version. 
+
+{{< highlight shell >}}
+$ sensuctl asset add sensu/sensu-influxdb-handler:3.1.1
```

Comment:
Do we want the `$` removed? I believe we have moved towards our CLI examples not having it.